docs/_docs/step-by-step/10-deployment.md

```diff
@@ -42,6 +42,11 @@ bundle exec jekyll serve
 
 This restricts your Ruby environment to only use gems set in your `Gemfile`.
 
+Note: if publishing your site with GitHub Pages, you can match production
+version of Jekyll by using the `github-pages` gem instead of `github-pages`
```

Comment:
> by using the `github-pages` gem instead of `github-pages`
in your `Gemfile`

I... don't understand what this is saying?

Reply:
Thank you, fixed at 2621159fc0301ca773f32886ced6c87cc3398b48